src/index.js

```diff
@@ -638,125 +649,121 @@ const members = {
 
         this.appendChild(firstChild);
       }
-    }
+    },
   },
   insertBefore: {
-    value (newNode, refNode) {
+    value(newNode, refNode) {
       return appendChildOrInsertBefore(this, newNode, refNode);
-    }
+    },
   },
   lastChild: {
-    get () {
+    get() {
       const ch = this.childNodes;
       return ch[ch.length - 1] || null;
-    }
+    },
   },
   lastElementChild: {
-    get () {
+    get() {
       const ch = this.children;
       return ch[ch.length - 1] || null;
-    }
+    },
   },
   name: {
-    get () {
+    get() {
       return this.getAttribute('name');
     },
-    set (name) {
+    set(name) {
       return this.setAttribute('name', name);
-    }
+    },
   },
   nextSibling: {
-    get () {
+    get() {
       const host = this;
-      return eachChildNode(this.parentNode, function (child, index, nodes) {
+      return eachChildNode(this.parentNode, (child, index, nodes) => {
         if (host === child) {
           return nodes[index + 1] || null;
         }
       });
-    }
+    },
   },
   nextElementSibling: {
-    get () {
+    get() {
       const host = this;
       let found;
-      return eachChildNode(this.parentNode, function (child) {
+      return eachChildNode(this.parentNode, (child) => {
         if (found && child.nodeType === 1) {
           return child;
         }
         if (host === child) {
           found = true;
         }
       });
-    }
+    },
   },
   outerHTML: {
-    get () {
+    get() {
       const name = this.tagName.toLowerCase();
-      const attributes = Array.prototype.slice.call(this.attributes).map(function (attr) {
-        return ` ${attr.name}${attr.value ? `="${attr.value}"` : ''}`;
-      }).join('');
+      const attributes = Array.prototype.slice.call(this.attributes).map((attr) => (` ${attr.name}${attr.value ? `="${attr.value}"` : ''}`)).join(''); // eslint-disable-line max-len
```

Comment:
i think the old `attributes` assignment was much more readable. would be good to split the map function onto a new line, not sure having it all on the same line is worth it.